reference/forms/types/birthday.rst

```diff
@@ -1,13 +1,13 @@
 .. index::
    single: Forms; Fields; birthday
 
-birthday Field Type
+Birthday Field Type
```

Comment:
I don't really follow why we lowercased the name here each time. Do you agree with capitializing them @symfony/team-symfony-docs ?